app/controllers/concerns/idv_session.rb

```diff
@@ -26,16 +28,23 @@ def idv_session
   protected
 
   def idv_attempts=(num)
-    user_session[:idv_attempts] = num
+    current_user.update!(idv_attempts: num)
   end
 
   def idv_attempts
-    user_session[:idv_attempts] ||= 0
+    current_user.idv_attempts ||= 0
+  end
```

Comment:
Now that the attempts are stored in the DB, the controller no longer needs these getter and setter methods, right? Any time the attempts need to be set to a particular value, wouldn't it be done on the user directly? The getters and setters are redundant now that you can call `current_user.update`.